marimo/_server/file_manager.py

```diff
@@ -8,28 +8,35 @@
 from marimo._ast import codegen
 from marimo._ast.app import App, InternalApp, _AppConfig
 from marimo._ast.cell import CellConfig
+from marimo._config.config import MarimoConfig
 from marimo._runtime.layout.layout import (
     LayoutConfig,
     read_layout_config,
     save_layout_config,
 )
 from marimo._server.api.status import HTTPException, HTTPStatus
-from marimo._server.models.models import (
-    SaveRequest,
-)
+from marimo._server.models.models import SaveRequest
 from marimo._server.utils import canonicalize_filename
 
 LOGGER = _loggers.marimo_logger()
 
 
 class AppFileManager:
-    def __init__(self, filename: Optional[str]) -> None:
+    def __init__(
+        self,
+        filename: Optional[str],
+        user_config: MarimoConfig | None = None,
+    ) -> None:
         self.filename = filename
+        # If provided, user_config overrides default app config for new apps
+        self._user_config = user_config
         self.app = self._load_app(self.path)
 
     @staticmethod
-    def from_app(app: InternalApp) -> AppFileManager:
-        manager = AppFileManager(None)
+    def from_app(
+        app: InternalApp, user_config: MarimoConfig | None = None
```

Comment:
if you change the constructor to just take in `initial_width`, then you dont need to pass `user_config: MarimoConfig | None = None` since it will have the width on it

`from_app` should already have `width`

Reply:
Will change from staticmethod to a regular method

Reply:
i dont think you need to do that - i was just thinking the app already have the `width` on it, which you can pass into the constructor